PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib-java/src/main/java/com/microsoft/azure/toolkit/intellij/connector/completion/provider/AbstractResourceConnectionCompletionProvider.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.azure.toolkit.intellij.connector.completion.provider;
+
+import com.intellij.codeInsight.completion.CompletionParameters;
+import com.intellij.codeInsight.completion.CompletionProvider;
+import com.intellij.codeInsight.completion.InsertionContext;
+import com.intellij.codeInsight.lookup.LookupElement;
+import com.intellij.codeInsight.lookup.LookupElementBuilder;
+import com.intellij.openapi.command.WriteCommandAction;
+import com.intellij.openapi.module.Module;
+import com.microsoft.azure.toolkit.ide.common.icon.AzureIcons;
+import com.microsoft.azure.toolkit.intellij.common.IntelliJAzureIcons;
+import com.microsoft.azure.toolkit.intellij.connector.*;
+import com.microsoft.azure.toolkit.intellij.connector.completion.model.CompletionItem;
+import com.microsoft.azure.toolkit.lib.common.model.AzResource;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+
+import javax.annotation.Nonnull;
+import javax.swing.*;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+public abstract class AbstractResourceConnectionCompletionProvider extends CompletionProvider<CompletionParameters> {
+    public static final String INDICATOR = "IntellijIdeaRulezzz ";
```

Comment:
`*zzz`?

Reply:
This is the indicator for code actions for intellij, refers https://github.com/JetBrains/intellij-community/blob/master/platform/core-api/src/com/intellij/codeInsight/completion/CompletionUtilCore.java#L7-L12